app/src/main/java/edu/grinnell/appdev/grinnelldirectory/models/SimpleResult.java

```diff
@@ -0,0 +1,36 @@
+package edu.grinnell.appdev.grinnelldirectory.models;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import java.util.List;
+
+public class SimpleResult implements Parcelable {
```

Comment:
This is a class that should be tested before using! Could you please write an androidTest class for this that tests all parts of this class that we will be using? For example, create a list of random `Person`s, and check if it parcels it without error. Then recreate the list (un-parcel), and compare that object to the original list @as-f